src/calchartdoc.cpp

```diff
@@ -279,6 +281,25 @@ wxInputStream& CalChartDoc::LoadObject(wxInputStream& stream)
 }
 #endif
 
+bool CalChartDoc::exportViewerFile(const wxString& filepath) {    
+    JSONElement mainObjectElement = JSONElement::makeObject();
+    
+    JSONDataObjectAccessor mainObject = mainObjectElement;
```

Comment:
just curious, why do you rename it to `mainObject` instead of using `mainObjectElement` directly?